src/app/core/sync-status/background-processing-indicator/background-processing-indicator.component.ts

```diff
@@ -22,7 +22,7 @@ export class BackgroundProcessingIndicatorComponent implements OnChanges {
   @Input() backgroundProcesses: BackgroundProcessState[] = [];
 
   /** whether processes of with the same title shall be summarized into one line */
-  @Input() summarize: boolean = true;
+  @Input() summarize: boolean = false;
```

Comment:
I dont think this should go into the production mode because the information is not necessary for the final user

Reply:
Funny side note: This was the actual reason that the search bar wasn't working (resp. the `summarizeProcesses` function) since it deleted something from the array which caused the `queryIndexRaw` function to wait forever)